docs/_embed/solutions/partner-api.md

```diff
@@ -243,93 +243,119 @@ You also cannot prefill an action field with data returned from the trigger. In
-|  **redirect_uri**   |  Required   | The URI you provided in the sign-up form. If you need to modify this, you'll need to [send us a request](mailto:partners@zapier.com).                                                                               |
-|  **response_type**  |  Required   | Use `token`.                                                                                                                                                                                                        |
-|      **scope**      |  Optional   | Space (`%20`) separated values. See each resource for their required scope, if any. This (and all other params) should be properly [url encoded](https://en.wikipedia.org/wiki/Percent-encoding).                   |
-| **approval_prompt** |  Optional   | One of `auto` or `force`. Use `auto` if the second authorization (before expiration of previous token) should not prompt the user to re-authorize. Use `force` if the user should authorize your application again. |
-|      **state**      |  Optional   | A unique string to help your application guard against XSRF.                                                                                                                                                        |
 
-**Example Prompt**
+You can configure one (or multiple) **redirect URIs** in the [Zapier Developer Platform](https://developer.zapier.com/) under `Embed`→`Settings`→`Redirect URIs`
 
-![Example OAuth2 Authorization Prompt](https://cdn.zapier.com/storage/photos/d926ea5ba6ca80c184a45cf3f5e420fb.png)
+![Configure redirect URIs](https://cdn.zappy.app/f8fbb9cf76864f457a0132b7eb8db196.png)
 
-##### Receiving the token, or error
+##### Prerequisite 2: Get your Client ID and Client Secret
+>Your application's **Client ID** and **Client Secret** are only available after you've published your app as a [public integration in Zapier's App Directory](https://platform.zapier.com/quickstart/private-vs-public-integrations).
 
-If the user cancels, or approves the authorization the user will be redirected to your `redirect_uri` with the following example urls:
+You can find your **Client ID** and **Client Secret** in the [Zapier Developer Platform](https://developer.zapier.com/) under `Embed`→`Settings`→`Credentials`
 
-**Approved**
-
-```
-http://your.redirect.url/#access_token=iuqhw8egojqenduvybtoken_type=Bearer&expires_in=36000&scope=zap
-```
+![Client ID and Secret](https://cdn.zappy.app/cb3660c17a3d26b36f438ab80c0860d5.png)
 
-**Cancelled**
+---
 
+##### 1. Determine which OAuth scopes are required for your use case
+The various endpoints of the Zapier Partner API require different OAuth scopes. See the [API Reference](/api-reference/actions/get-actions) for documentation regarding which OAuth scopes are required by which endpoints.
+![OAuth Scope documentation example](https://cdn.zappy.app/3667a31edfb56bc1187739d8b1303c5c.png)
```

Comment:
This link is broken since it points to a Mintlify path, and the screenshot is also of Mintlify which is not yet live. Should we perhaps link to https://docs.api.zapier.com/api/ or something else that is live, and change/remove the screenshot?

Reply:
Good catch, @cyberwitch. 

Here's a new link to the stoplight docs: https://docs.api.zapier.com/api#/7b2a64020b290-get-actions

And a screenshot from the stoplight docs: https://cdn.zappy.app/9eb87de28fe59000353b78d594dad17e.png